src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -65,20 +101,35 @@ const ProfilePicture = ({
 
 let styles = StyleSheet.create({
   addCircle: {position: 'absolute', top: 25, left: 25},
-  profileImage: (size, width) => ({
-    width: dimen.normalizeDimen(size * 0.91),
-    height: dimen.normalizeDimen(size * 0.91),
-    borderRadius: 100,
-    marginLeft: dimen.normalizeDimen(width / 2),
-    marginTop: dimen.normalizeDimen(width / 2)
-  }),
+  profileImage: (size) => {
+    const scale = size <= 25 ? 0.8 : 0.92;
+    const marginScale = size <= 25 ? 0.805 : 0.925;
+    return {
+      width: dimen.normalizeDimen(size * scale),
+      height: dimen.normalizeDimen(size * scale),
+      borderRadius: 100,
+      marginHorizontal: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
+      marginVertical: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2
+    };
+  },
+  anonStyle: (size) => {
+    const scale = size <= 25 ? 0.8 : 0.92;
+    const marginScale = size <= 25 ? 0.805 : 0.925;
+    return {
+      width: dimen.normalizeDimen(size * scale),
+      height: dimen.normalizeDimen(size * scale),
+      borderRadius: 100,
+      marginHorizontal: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
+      marginVertical: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
+      justifyContent: 'center',
+      alignItems: 'center'
+    };
+  },
   profileImageContainer: (size) => ({
     width: size,
-    borderRadius: 100
-  }),
-  wrapImageProfile: {
-    flexDirection: 'column'
-  }
+    borderRadius: 100,
+    height: size
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles for `profileImage` and `anonStyle` are almost identical, which violates the DRY (Don't Repeat Yourself) principle. Consider creating a common style that can be used by both, and then extend it as necessary for each specific case.
```diff
-  profileImage: (size) => {
-    const scale = size <= 25 ? 0.8 : 0.92;
-    const marginScale = size <= 25 ? 0.805 : 0.925;
-    return {
-      width: dimen.normalizeDimen(size * scale),
-      height: dimen.normalizeDimen(size * scale),
-      borderRadius: 100,
-      marginHorizontal: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
-      marginVertical: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2
-    };
-  },
-  anonStyle: (size) => {
-    const scale = size <= 25 ? 0.8 : 0.92;
-    const marginScale = size <= 25 ? 0.805 : 0.925;
-    return {
-      width: dimen.normalizeDimen(size * scale),
-      height: dimen.normalizeDimen(size * scale),
-      borderRadius: 100,
-      marginHorizontal: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
-      marginVertical: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
-      justifyContent: 'center',
-      alignItems: 'center'
-    };
-  },
+  commonStyle: (size) => {
+    const scale = size <= 25 ? 0.8 : 0.92;
+    const marginScale = size <= 25 ? 0.805 : 0.925;
+    return {
+      width: dimen.normalizeDimen(size * scale),
+      height: dimen.normalizeDimen(size * scale),
+      borderRadius: 100,
+      marginHorizontal: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2,
+      marginVertical: (dimen.normalizeDimen(size) - dimen.normalizeDimen(size * marginScale)) / 2
+    };
+  },
+  profileImage: (size) => ({
+    ...commonStyle(size)
+  }),
+  anonStyle: (size) => ({
+    ...commonStyle(size),
+    justifyContent: 'center',
+    alignItems: 'center'
+  }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->